experiments/tf_trainer/tf_hub_tfjs/notebook/BiasEvaluation.ipynb

```diff
@@ -0,0 +1,512 @@
+      "provenance": [],
+      "collapsed_sections": []
+    },
+    "kernelspec": {
+      "name": "python3",
+      "display_name": "Python 3"
+    },
+    "accelerator": "GPU"
+  },
+  "cells": [
+    {
+      "metadata": {
+        "id": "9B7PdsrvW__k",
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "# Bias Evaluation for TF Javascript Model\n",
+        "\n",
+        "Based on the [FAT* Tutorial Measuring Unintended Bias in Text Classification Models with Real Data](https://github.com/conversationai/unintended-ml-bias-analysis/blob/master/presentations/FAT_star_tutorial.md).\n",
+        "\n",
+        "Copyright 2019 Google LLC.\n",
+        "SPDX-License-Identifier: Apache-2.0"
+      ]
+    },
+    {
+      "metadata": {
+        "id": "0Jsjp3E5rbuC",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "cell_type": "code",
+      "source": [
+        "!pip3 install --quiet \"tensorflow>=1.11\"\n",
+        "!pip3 install --quiet sentencepiece"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "metadata": {
+        "id": "4bSQf93oVo7j",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "cell_type": "code",
+      "source": [
+        "from __future__ import absolute_import\n",
+        "from __future__ import division\n",
+        "from __future__ import print_function\n",
+        "\n",
+        "import pandas as pd\n",
+        "import numpy as np\n",
+        "import matplotlib.pyplot as plt\n",
+        "import seaborn as sns\n",
+        "import tensorflow as tf\n",
+        "import sentencepiece\n",
+        "from google.colab import auth\n",
+        "from IPython.display import HTML, display\n",
+        "\n",
+        "from sklearn import metrics\n",
+        "\n",
+        "%matplotlib inline\n",
+        "\n",
+        "# autoreload makes it easier to interactively work on code in imported libraries\n",
+        "%load_ext autoreload\n",
+        "%autoreload 2\n",
+        "\n",
+        "# Set pandas display options so we can read more of the comment text.\n",
+        "pd.set_option('max_colwidth', 300)\n",
+        "\n",
+        "# Seed for Pandas sampling, to get consistent sampling results\n",
+        "RANDOM_STATE = 123456789"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "metadata": {
+        "id": "12LU1AjWr-da",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "cell_type": "code",
+      "source": [
+        "auth.authenticate_user()"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "metadata": {
+        "id": "I4dzfYrvr3XT",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "cell_type": "code",
+      "source": [
+        "# Download and unzip files used in this colab\n",
+        "!curl -O -J -L https://storage.googleapis.com/civil_comments/fat_star_tutorial/fat-star.zip\n",
```

Comment:
I believe the only file you are using from fat-star.zip is public_test.csv.  It might be good to just copy that file to the `gs://conversationai-public/public_models/tfjs/v1/` directory, that way you aren't downloading extra stuff not needed here, and if we ever changed the FAT* colab we wouldn't risk affecting this colab

Reply:
Now using madlibs